test/scripts/e2e_subs/hdr-access.py

```diff
@@ -85,11 +81,33 @@
  int 0
  block BlkTimestamp
 """
-tx = goal.app_create(joe, goal.assemble(teal))
-tx.last_valid_round = tx.last_valid_round - 800
-txinfo, err = goal.send(tx)
+txinfo, err = goal.app_create(joe, goal.assemble(teal), lifetime=100)
 assert "round 0 is not available" in str(err), err
 assert "outside [1-" in str(err), err  # confirms that we can look back to 1
 
+
+# Get FeeSink from `block` opcode, compare to REST API
+teal = """
+#pragma version 11
+ txn FirstValid
+ int 2
+ -
+ block BlkFeeSink
+ log
+ int 1
+ return
+"""
+txinfo, err = goal.app_create(joe, goal.assemble(teal), lifetime=100)
+assert not err, err
+assert len(txinfo["logs"]) == 1
+opcode = txinfo["logs"][0]
+
+block = goal.algod.block_info(txinfo['confirmed-round']-2)['block']
+api = base64.b64encode(enc.decode_address(block['fees'])).decode("utf-8")
+
+print(opcode, api)
```

Comment:
forgotten debug print?

Reply:
Yes and no. Python's assert does not print the objects being compared, so this is helpful in the case of failure. When the test succeeds, the output is surpressed anyway.

I'll remove it if you'd prefer it gone, let me know.